packages/editor/src/lib/TldrawEditor.tsx

```diff
@@ -516,3 +524,118 @@ export function useOnMount(onMount?: TLOnMountHandler) {
 		if (editor) return onMountEvent?.(editor)
 	}, [editor, onMountEvent])
 }
+
+const WATERMARK_SRC = `url('data:image/svg+xml;utf8,${encodeURIComponent(watermarkDesktopSvg)}') center 100% / 100% no-repeat`
+
+const Watermark = React.memo(() => {
+	const events = useCanvasEvents()
+
+	const ref = useRef<HTMLAnchorElement>(null)
+	useLayoutEffect(() => {
+		// eslint-disable-next-line deprecation/deprecation
+		if (ref?.current) ref.current.style.webkitMask = WATERMARK_SRC
+	}, [])
+
+	const editor = useEditor()
+	const showWatermark = useValue(
+		'show watermark',
+		() => {
+			const { width } = editor.getViewportScreenBounds()
+			if (width < 760) return false
+			if (!featureFlags.enableLicensing.get()) return false
+			return editor.getLicenseState() === 'unlicensed'
+		},
+		[editor]
+	)
+	const isDebugMode = useValue('debug mode', () => editor.getInstanceState().isDebugMode, [editor])
+	const isMenuOpen = useValue('is menu open', () => editor.getIsMenuOpen(), [editor])
+
+	if (!showWatermark) return null
+
+	const className = LicenseManager.classname
+	const imageClassName = LicenseManager.classname + lns(`watermark${version.replace(/\./g, '')}`)
```

Comment:
`lns` would work here but you shouldn't put the `LicenseManager.classname` prefix on it, otherwise it's too easy to query select on. here's why:

you can just find that easily with an attribute selector (that looks for that string in the attribute:
`'[class^="tl-watermark_SEE-LICENSE"]'` or `'[class*="tl-watermark_SEE-LICENSE"]'`

Reply:
ok, will remove this one

Reply:
That whole thing feels unnecessary tbh, binning it